<launch>

    <node pkg="markertracker_node" type="process_frame_topic.py" name="markertracker_node" output="screen">

        <param name="input_image_topic" type="string" value="raspicam_node/image_raw"/>
        <param name="publish_topic_result" value="True"/>
        <rosparam param="marker_ids" subst_value="True">[10,72]</rosparam>

    </node>

</launch>